### ISSUE TITLE
Update the rack gem

### ISSUE THREAD

**Issue by rlgreen91** [EBWiki/EBWiki#1672]
Update the `rack` gem to the highest version possible given that the system employs Ruby 2.4.1, Rails 4.2.10, and PostGres 9.4.  Be sure to pin the major and minor version in the Gemfile (e.g., if a gem my_gem is at 1.5.2, the Gemfile should read `gem my_gem, ~> 1.5`).

**Comment by StlMaris123** [comment]
Hi @rlgreen91 , looks like this will have to wait till we upgrade rails
```
 rails (~> 4.2.10) was resolved to 4.2.10, which depends on
      actionpack (= 4.2.10) was resolved to 4.2.10, which depends on
        rack (~> 1.6)
```

**Comment by rlgreen91** [comment]
Sounds good.  Since this is dependent on Rails, I'll go ahead and close this.
